servicetest/filesystem/test_filesystem_strace.py

```diff
@@ -0,0 +1,191 @@
+# Copyright (C) 2016-2017 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+
+import pytest
+
+import os
+import time
+import platform
+import socket
+import re
+import StringIO
+
+
+from testframework import Container
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+TESTOUTPUT_DIR = CURRENT_DIR + "/testoutput"
+TESTFILE = 'testfile.txt'
+
+
+# This function is used by the test framework to know where test specific
+# files should be stored
+def output_dir():
+    return TESTOUTPUT_DIR
+
+
+# This function is used by the 'agent' fixture to know where the log should
+# be stored
+def logfile_path():
+    return CURRENT_DIR + "/test-strace.log"
+
+def agent_exec_prefix():
+    return "strace"
+
+DATA = {
+    Container.CONFIG: '[{"enableWriteBuffer": false}]',
+    Container.BIND_MOUNT_DIR: "/gateways/app",
+    Container.HOST_PATH: CURRENT_DIR,
+    Container.READONLY: False
+}
+
+def get_logblock(file, startline, stopline):
+    """ This gets a block of text/logs between the startline and stopline
+    from a file and returns the text as a string to the caller. Does not
+    include startline or stopline in the actual textblock passed to the caller.
+
+    :param file: The file to read
+    :param startline: The first line to look for
+    :param stopline: The last line to look for
+    :return: Returns a textbuffer
+    """
```

Comment:
Please make indentation and comment style in the same way as in the other parts of the code (the other modules).